From 08f66ad49d3d0b9e50de17b01f15c20bdce8ea30 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Thu, 5 Mar 2015 10:13:43 +0800
Subject: [PATCH 0869/1594] MLK-10371: ov5640: Rename int-device ov5640 driver
 to ov564x

commit a2d4b432d081c151b19da98813f5b787fbff0dc8 from
git://git.freescale.com/imx/linux-2.6-imx.git

There are two ov5640 driver in kernel, one is subdev ov5640 driver,
the other is int-device ov5640 driver.
Rename int-device ov5640 driver to ov564x.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 drivers/media/platform/mxc/capture/ov5640.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/mxc/capture/ov5640.c b/drivers/media/platform/mxc/capture/ov5640.c
index 2f7253c..ec8b809 100644
--- a/drivers/media/platform/mxc/capture/ov5640.c
+++ b/drivers/media/platform/mxc/capture/ov5640.c
@@ -575,7 +575,7 @@ MODULE_DEVICE_TABLE(i2c, ov5640_id);
 static struct i2c_driver ov5640_i2c_driver = {
 	.driver = {
 		  .owner = THIS_MODULE,
-		  .name  = "ov5640",
+		  .name  = "ov564x",
 		  },
 	.probe  = ov5640_probe,
 	.remove = ov5640_remove,
@@ -1797,7 +1797,7 @@ static struct v4l2_int_slave ov5640_slave = {
 
 static struct v4l2_int_device ov5640_int_device = {
 	.module = THIS_MODULE,
-	.name = "ov5640",
+	.name = "ov564x",
 	.type = v4l2_int_type_slave,
 	.u = {
 		.slave = &ov5640_slave,
-- 
1.7.5.4

